src/components/user/uc/users.uc.js

```diff
@@ -125,6 +115,22 @@ const checkPermissions = async (id, roleName, permissionAction, { user: currentU
 	}
 };
 
+/**
+ * Checks if the user defined by params.user has the right to perform the action defined in permissionAction on the user defined in id
+ *
+ * @param {string|string[]} affectedUserId ID or list of IDs of the user(s) to be affected by the request
+ * @param {*} roleName the role name that the effected user should have
+ * @param {*} permissionAction the action that is to be performed (CREATE, EDIT, DELETE)
+ * @param {Object} params needs to contain a property `user` with populated roles and permissions, which is the user issuing the request
+ */
+const checkPermissions = async (affectedUserId, roleName, permissionAction, { user: currentUser }) => {
+	const ids = Array.isArray(affectedUserId) ? affectedUserId : [affectedUserId];
+	const affectedUsers = await userRepo.getUsersWithRoles(ids);
+	return Promise.all(
```

Comment:
No `Promise.all` needed anymore when `checkPermissionsInternal` is syncronous.

Reply:
ok, thanks. replaced by forEach()